### ISSUE TITLE
Question about Neural Prophet performance

### ISSUE THREAD

**Issue by landkwon94** [ourownstory/neural_prophet#1426]
Hello! I asked several times before, and now I am writing manuscripts using Neural Prophet model for cropland monitoring :) 

Thank you so much in advance!

By the way, I have a question about the performance of this model.

In my case, I want to mainly forecast peak season of target variable, using 6 lagged regressors.

Because those peak values are determined by regressor values from the other days.

It seems Neural Prophet model is training the model in the whole days.

So my question is that, is Neural Prophet possible to train in the specific season and forecast in the other specific season?

I attached a figure below for better understanding!

Thank you again for the nice model!

![그림4](https://github.com/ourownstory/neural_prophet/assets/73383617/d114552d-2e30-4b4b-a8a4-6dbbfad0bb29)

**Comment by leoniewgnr** [comment]
Yes this is possible and this is also how it is intended! Very cool use case!
So what you can do, split the dataset into train and test set (either with `m.split_df` or with pandas directly)
And then train on the first set:
`m.fit(df_train)`
and then test/predict on the test set:
`metrics = m.predict(df_test)`

You can see an example in this tutorial: https://neuralprophet.com/tutorials/tutorial10.html
